python/ray/autoscaler/_private/kuberay/run_autoscaler.py

```diff
@@ -96,9 +96,11 @@ def _setup_logging() -> None:
     # The director should already exist, but try (safely) to create it just in case.
     try_to_create_directory(log_dir)
 
+    logging_level = ray_constants.AUTOSCALER_LOG_LEVEL
+
     # Write logs at info level to monitor.log.
     setup_component_logger(
-        logging_level=ray_constants.LOGGER_LEVEL,  # info
```

Comment:
I wonder if we should make this ray_constants.LOGGER_LEVEL a configurable thing through env actually.